tutorials/single_node_tutorial/single_gpu_tutorial.ipynb

```diff
@@ -306,7 +306,7 @@
    "source": [
     "res = download_wikipedia(download_output_directory,\n",
     "                   language=language, \n",
-    "                   url_limit=url_limit).df.compute()"
+    "                   url_limit=url_limit, dump_date=\"20250201\").df.compute()"
```

Comment:
I left a comment on the issue too, but I cannot reproduce this error. The most recent dump date (20250220) works for me.

Let me know what you think, thanks!

Reply:
Yup I cannot reproduce as well. This isn't an issue anymore.